oauthproxy.go

```diff
@@ -838,6 +839,44 @@ func (p *OAuthProxy) enrichSessionState(ctx context.Context, s *sessionsapi.Sess
 func (p *OAuthProxy) AuthOnly(rw http.ResponseWriter, req *http.Request) {
 	session, err := p.getAuthenticatedSession(rw, req)
 	if err != nil {
+		if authOnlyRedirectSignIn(req) {
+			logger.Printf("Initiating AuthOnly SignIn Redirect")
```

Comment:
I don't think we would want this logged in production, please remove before we merge

Reply:
removed